packages/relative-values/models/src/animals.ts

```diff
@@ -0,0 +1,617 @@
+import { Model } from "@/model/utils";
+import { Catalog, InterfaceWithModels, Item } from "@/types";
+import { sq } from "@quri/squiggle-lang";
+
+// Note: Much of this was generated by this file:
+// https://observablehq.com/d/98cbe0073226a5f9
+
+function getCatalog(): Catalog {
+  const items: Item[] = [
+    {
+      id: "cow_value_human_qalys_per_year_of_life",
```

Comment:
Why is "human" in here? We're using relative values. People will only see this vs. human QALYs, when looking at the value vs. a "human_QALY" Item (which you don't have now, but could add)

Reply:
Yep, makes sense. It's there because I used human qalys to keep track, though this could later change.

Reply:
The fact that you used human qalys in your model is very much an internal detail to your model, it shouldn't effect these IDs. 

That said, this seems kind of useless if you don't even include human qalys as an Item (though I guess you're expecting to add that later).